src/SearchQueryService/Indexes/IndexesProcessor.cs

```diff
@@ -84,7 +85,8 @@ private async Task<int> GetSchemaSize(string indexName)
             StringContent data = new(postJson, Encoding.UTF8, "application/json");
             var indexUrl = _connectionStrings["Solr"].AppendPathSegments(indexName, "schema");
 
-            await _httpClient.PostAsync(indexUrl, data);
+            var test = await _httpClient.PostAsync(indexUrl, data);
+            var content = test.Content.ReadAsStringAsync();
```

Comment:
Potrebuješ to tu naozaj čítať? Alebo si to len používal, keď si to ladil la zabudol si to odstrániť?

Reply:
Ten PR som robil narýchlo pred odchodom a už som si ho neskontroloval 😅